server/pulp/server/async/scheduler.py

```diff
@@ -283,10 +285,14 @@ def tick(self):
 
             except mongoengine.NotUniqueError:
                 # Setting a default wait time for celerybeat instances with no lock
-                ret = constants.CELERY_TICK_DEFAULT_WAIT_TIME
-                _logger.info(_("Duplicate or new celerybeat Instance, "
-                               "ticking again in %(ret)s seconds.")
-                             % {'ret': ret})
+                ret = constants.PULP_PROCESS_HEARTBEAT_INTERVAL
+
+                if not self._retry_msg_logged:
+                    _logger.info(_("Duplicate or new celerybeat Instance, "
+                                   "ticking again in %(ret)s seconds.")
+                                 % {'ret': ret})
+                    # Never re-log this message
+                    self._retry_msg_logged = True
```

Comment:
What if this variable was renamed from `_retry_msg_logged` to `_already_logged`? The if state would read more like plaintext and the comment at L#294 could be removed. Really it's the comment that got me interested in this. @ehelms taught me to reserve comment use for documenting programmer intent that is significantly non-obvious. I can rant more about the evil of comments in code some other time...